pkgs/tools/misc/memtest86+/default.nix

```diff
@@ -1,31 +1,35 @@
 { lib, stdenv, fetchgit }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "memtest86+";
-  version = "5.01-coreboot-002";
+  version = "6.00-beta2";
 
   src = fetchgit {
-    url = "https://review.coreboot.org/memtest86plus.git";
-    rev = "v002";
-    sha256 = "0cwx20yja24bfknqh1rjb5rl2c0kwnppzsisg1dibbak0l8mxchk";
+    url = "https://github.com/memtest86plus/memtest86plus";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-U3++iJa0Zj3g2SZTJ0jom7raAu+LGqiOKZEputs/YfM=";
   };
 
-  NIX_CFLAGS_COMPILE = "-I. -std=gnu90";
+  # Binaries are booted directly by BIOS/UEFI or bootloader
+  # and should not be patched/stripped
+  dontPatchELF = true;
+  dontStrip = true;
 
-  hardeningDisable = [ "all" ];
+  passthru.efi = "${finalAttrs.finalPackage}/memtest.efi";
```

Comment:
Can we use `${placeholder "out"}` here? finalAttrs has still performance problems.

Reply:
If someone uses .override or .overrideAttrs with the current code it'll work, if I change it to the old way it will still point to the old path.

Is it not actually ready for use yet?

cc @roberth

Reply:
last time I checked finalAttrs had some major performance issues which needed to be solved.

Reply:
Asked roberth on matrix and sounds like it should be fine here, as it's not adding another .overrideAttrs call.

The way I did it in #176312 isn't good though.